extensions/wikia/ArticleVideo/scripts/ooyala-player.js

```diff
@@ -126,7 +126,23 @@ define('ooyala-player', function () {
 						tag_url: vastUrl
 					}
 				],
-				useGoogleCountdown: true
+				useGoogleAdUI: true,
+				useGoogleCountdown: false,
+				onBeforeAdsManagerStart: function (IMAAdsManager) {
+					// mutes VAST ads from the very beginning
+					// FIXME with VPAID it causes volume controls to be in incorrect state
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)